### PR TITLE
Rodeo ingestion

### LINKED ISSUE #35: Bounty → Rodeo.Club Mint Ingestor
* Generate a `MintTemplate` that supports a [rodeo.club](https://rodeo.club/) mint on Base from a URL
* Bounty for this ingestor is 500 USDC, paid upon:
  * Successful approval & merge of the pull request
  * Ingestion of one mint into Mobile Minting ✨

### PR DESCRIPTION
**Pull request by Myestery**
### Functionality Supported
* Ingesting from URL: Yes 
* Ingesting from Contract address: Yes 
* Supported Networks: Base 

### Before you submit
- [x] Ensure your generated MintTemplate works 😄
- [x] Ensure that your code is restricted to a single folder in `src/ingestors`
- [x] Ensure that all required assets are included (e.g. ABIs)
- [x] Ensure ABIs are trimmed to include only methods (1) used in the ingestor or (2) required to mint
- [x] Ensure that all exported methods are prefixed with the name of your ingestor e.g. `myMintingPlatformGetContractDetails`
- [x] Ensure that a test exists for generating a MintTemplate that will always succeed
- [x] Ensure that your code accesses **no external resources** except those passed in the `resources` object

fix #35

### CODE REVIEW

**Comment by chrismaddern** [comment]
Hey @Myestery 👋 

Excited to see you hopping in on another one!!

Just a headsup that we've gotten a few questions from other folks wanting to jump in here too, so for incomplete submissions we'll only hold the bounty for the first submitter for a few days.

Just wanted to give you a heads up since you were first, but it's not built yet.

**Comment by Myestery** [comment]
oh yeah
I understand that and would turn this into a real PR tomorrow
I already worked on it during the weekend

File: test/ingestors/rodeo.test.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    resources,
+    {
+      successUrls: [
+        'https://rodeo.club/post/0x68227a4390c15AcEf9265d9B8F65d3fb5cD9f85B/1',
+        'https://rodeo.club/post/0x6511cB5ec4dbe28a6F2cbc40d2d1030b6CaBC911/10',
+        'https://rodeo.club/post/0x98E9116a26E1cf014770122b2f5b7EE4Cad067bA/1?utm_source=twitter&utm_medium=tweet&utm_campaign=hot_ones',
+      ],
+      failureUrls: ['https://rodeo.club/post/0x68227a4390c15AcEf9265d9B8F65d3fb5cD9f85', 'https://www.transient.xyz/stacks'],
+      successContracts: [
+        // { chainId: 8453, contractAddress: '0x6511cB5ec4dbe28a6F2cbc40d2d1030b6CaBC911' },
```
[the remark this answers is not in the document]
Comment (Myestery) [reply test/ingestors/rodeo.test.ts:22]: I ran into some issues but I'll fix the tests soon

**Comment by Myestery** [comment]
Hello @christinehall  This PR should ready for merge now
I've handled every issue that there was

**Comment by Myestery** [comment]
Screenshot 2024-08-12 at 22 50 33

**Review by chrismaddern — APPROVED** [review]
Congratulations on a single-review accepted PR!!!

Great job! Would love to see more of these 🥳

**Comment by chrismaddern** [comment]
DMd you on twitter 😊

**Pull request merged**